### DISCUSSION TITLE
Decky Installer won't install on my Steamdeck

### DISCUSSION THREAD

**Discussion by EPC2016** [SteamDeckHomebrew/decky-loader#523] [Q&A]
The installation looks like this

chmod: cannot access '/home/deck/homebrew/services/PluginLoader': No such file or directory
Failed to connect to bus: No medium found
Failed to connect to bus: No medium found
Unit /etc/systemd/system/plugin_loader.service is masked, ignoring.
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
Failed to start plugin_loader.service: Unit plugin_loader.service is masked.
Failed to enable unit: Unit file /etc/systemd/system/plugin_loader.service is masked.
chown: cannot access '/home/deck/homebrew/services/*': No such file or directory

Another installation method shows like this

(deck@steamdeck ~)$ curl -L https://github.com/SteamDeckHomebrew/decky-installer/releases/latest/download/install_release.sh | sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
100  3413  100  3413    0     0   1025      0  0:00:03  0:00:03 --:--:--  3709
[sudo] password for deck: 
Installing Steam Deck Plugin Loader release...
Installing version ...
curl: no URL specified!
curl: try 'curl --help' for more information
chmod: cannot access '/home/deck/homebrew/services/PluginLoader': No such file or directory
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
Grabbed latest release service.
Failed to start plugin_loader.service: Unit plugin_loader.service is masked.
Failed to enable unit: Unit file /etc/systemd/system/plugin_loader.service is masked.

**Comment by NGnius** [comment]
> Failed to start plugin_loader.service: Unit plugin_loader.service is masked.
Failed to enable unit: Unit file /etc/systemd/system/plugin_loader.service is masked.

Looks like you already had it installed but disabled?

**Reply by EPC2016** [reply]
Everything was fine when I downloaded, but I went back to game mode without the addon, so I revisited the code to see what it was showing.
